### ISSUE TITLE
Plugins using `td-rs-derive-py` forced to depend on `td-rs-chop`

### ISSUE THREAD

**Issue by rosofo** [tychedelia/td-rs#13]
Guessing this is the lack of hygiene in macros at work: Compiler complains about the use of undeclared module `td_rs_chop` when trying to derive `PyOp`.

Workaround: adding `td-rs-chop = { path = "../../../td-rs-chop", features = ["python"] }` to the plugin fixes it

**Comment by rosofo** [comment]
relevant line: https://github.com/tychedelia/td-rs/blob/main/td-rs-derive-py/src/lib.rs#L238

**Comment by tychedelia** [comment]
Yeah, this is a bad design, basically wanted to avoid having to import both base + the op type libs, but that means we have to re-export base through the op type. Probably worth just spinning this out and requiring both or finding some other solution.
